test/plug/cast_test.exs

```diff
@@ -5,8 +5,28 @@ defmodule OpenApiSpex.Plug.CastTest do
 
   alias OpenApiSpexTest.ApiSpec
 
+  describe "valid operation (not annotated)" do
+    test "should work (with exdoc)" do
```

Comment:
Please rephrase this, "should work" is too broad and doesn't describe the expectation.
```elixir
test "does not raise in error when the operation is nil set with the operation macro"

test "does not raise in error when the operation is nil set with the open_api_operation function"
```

We shouldn't mention `exdoc` here, `ControllerSpecs-style operation specs` don't depend on ex_doc.

Reply:
Done. Fixed.